### PR TITLE
Create production Docker for Bassa UI with Nginx

### PR DESCRIPTION
**Pull request by kmehant**

### CODE REVIEW

File: ui/Dockerfile [hunk 1]
```diff
@@ -1,20 +1,14 @@
...
 
-COPY . .
-
-RUN apt-get update \
-&& npm install -g gulp \
-&& npm install -g bower \
-&& npm install --unsafe-perm
-
-CMD ["gulp", "serve"]
+RUN apt-get update
```
Comment (agentmilindu) [inline ui/Dockerfile]: Let's get this line before the COPY lines. Would be easy for layer caching.

**Comment by kmehant** [comment]
@agentmilindu

File: ui/Dockerfile [hunk 2]
```diff
@@ -1,20 +1,14 @@
-FROM node
+FROM nginx:alpine
 
 MAINTAINER SCoRe Lab Community <commuity@scorelab.org>
 
 ARG BUILD_DATE
 ARG VCS_REF
 
-WORKDIR /home/Bassa
+RUN apt-get update
```
Comment (agentmilindu) [inline ui/Dockerfile]: Alpine Linux does not use `apt-get` it seems, we have to replace this with an equivalent.

File: ui/Dockerfile [hunk 3]
```diff
@@ -1,20 +1,14 @@
...
-COPY . .
-
-RUN apt-get update \
-&& npm install -g gulp \
-&& npm install -g bower \
-&& npm install --unsafe-perm
-
-CMD ["gulp", "serve"]
+COPY nginx.conf /etc/nginx/conf.d/bassa.conf
+COPY ui/dist /var/www/bassa
```
Comment (agentmilindu) [inline ui/Dockerfile]: Should be `COPY dist /var/www/bassa`

**Pull request merged**